lib/src/signed_video_openssl.c

```diff
@@ -61,8 +61,8 @@ typedef struct {
  * OpenSSL cryptographic object.
  */
 typedef struct {
-  EVP_MD_CTX *primary_ctx;  // Hashing context
-  EVP_MD_CTX *secondary_ctx;  // Hashing context for fallback to gop level hashing.
+  EVP_MD_CTX *primary_ctx;  // Hashing context for the gop level hashing.
```

Comment:
It is not only used for GOP level hashing, it is also used for frame level hashing, since it is added to the SEI in both cases.